pom.xml

```diff
@@ -124,4 +165,83 @@
     </plugins>
   </reporting>
 
+  <profiles>
+    <profile> 
+      <id>release</id>
```

Comment:
According to the Maven lifecycle documentation at https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html the final lifecycle phase is "deploy" which is described as "copies the final package to the remote repository for sharing with other developers and projects". Should we model publishing to central as that phase instead? It seems more natural.